cea/demand/thermal_loads.py

```diff
@@ -179,7 +179,7 @@ def write_results(bpr, building_name, date, format_output, loads_output, locator
 
     # write report & quick visualization
     # (NOTE: uncomment to write full report for debugging purposes)
-    # cea.utilities.reporting.full_report_to_xls(tsd, locator.get_demand_results_folder(), building_name)
+    # reporting.full_report_to_xls(tsd, locator.get_demand_results_folder(), building_name)
```

Comment:
this should be handled via a flag in the config file or something. there is one called "debug" - we could either use that one or add a new one specific to reporting here like the `use_dynamic_infiltration_calculation`)

Reply:
I believe this is your code @daren-thomas. I was just fixing something that researchers here need.
I will create a new flag in the config file. Let me figure out how.

Reply:
if it's really just for developers, you can use the `debug` flag in the config file. that's what it's for :-)